### PR TITLE
SC-5500 - change wording Versuchen to lowercase

### PR DESCRIPTION
**Pull request by Grzesiek80**
# Description

Change wording Versuchen to lower case

## Links to Tickets or other pull requests
https://ticketsystem.schul-cloud.org/browse/SC-5500

## Screenshots of UI changes

2020-09-02_15h22_17

### CODE REVIEW

File: theme/n21/views/authentication/home.hbs [hunk 1]
```diff
@@ -28,7 +28,7 @@
                                     style="postion: absolute; top: 40%; font-size: 25px; position: absolute;"></i>
                                 <div class="placeholder" style="display: none;">
                                     Beim Laden des Inhalts der Startseite ist ein Fehler aufgetreten.
-                                    <button type="button" class="btn btn-primary mt-1">Erneut Versuchen</button>
+                                    <button type="button" class="btn btn-primary mt-1">Erneut versuchen</button>
```
Comment (mwolowyk-cap) [inline theme/n21/views/authentication/home.hbs:31]: why is internationalization label is not used here?

Comment (ABialoskorski) [reply theme/n21/views/authentication/home.hbs:31]: all this n21/home.hbs is not using internationalization, so I would leave it as it is

**Review by ABialoskorski — APPROVED** [review]

File: theme/n21/views/authentication/home.hbs (hunk 1 above)
Comment (Grzesiek80) [reply theme/n21/views/authentication/home.hbs:31]: because in this file all text are used in that way, I did not want to mix two approaches there

Comment (DanielRieckhof) [reply theme/n21/views/authentication/home.hbs:31]: Looks for me, that the usage of translations wasn't complete 
Maybe this is a follow up?

Comment (Grzesiek80) [reply theme/n21/views/authentication/home.hbs:31]: for that we should create another ticket and close it this one

**Review by mwolowyk-cap — APPROVED** [review]

**Review by DanielRieckhof — APPROVED** [review]

**Review by monaweitzenberg — APPROVED** [review]

**Pull request merged**